contracts/staking/contracts/test/TestStaking.sol

```diff
@@ -0,0 +1,53 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "../src/Staking.sol";
+
+
+contract TestStaking is
+    Staking
+{
+    // Stub out `payProtocolFee` to be the naive payProtocolFee function so that tests will
```

Comment:
Could we say a bit more here? I'm still not sure why tests would fail against the actual implementation.

Reply:
I ended up just removing it since they do work with the current impl